.github/workflows/deploy.yml

```diff
@@ -80,17 +100,22 @@ jobs:
 
       - uses: actions/upload-artifact@v2
         with:
-          name: pure-python-wheels.zip
+          name: pure-python-wheels
```

Comment:
Sorry if I'm getting confused, but also here isn't the `.zip` needed since we unzip the `.zip` files in line 117?

Reply:
The zip is implied in this instance. The artifacts are currently getting built as `.zip.zip` files. We can remove it from the name and the zip is still applied.